meilisearch-core/src/facets.rs

```diff
@@ -161,10 +171,9 @@ pub fn add_to_facet_map(
 ) -> Result<(), FacetError> {
     let value = match value {
         Value::String(s) => s,
-        Value::Object(_)
-        | Value::Array(_)
-        | Value::Number(_) => return Err(FacetError::InvalidDocumentAttribute(document_id)),
-        v => v.to_string(),
+        // ignore null
+        Value::Null => return Ok(()),
+        _ => return Err(FacetError::InvalidDocumentAttribute(document_id)),
```

Comment:
Would be great if you show the invalid field and the fact that we only support strings and arrays of strings only.